### ISSUE TITLE
Mobile: provider list/map, product list/grid

### ISSUE THREAD

**Issue by danrminds** [teamforus/forus#1048]
## Context/goal: 
https://www.dropbox.com/sh/xn5e8y7allspk32/AADXWwn_sQuBcuBObYgwxF5Wa?dl=0

- [x] Finish map view of providers and offers
    - [x] Map view
    - [x] Filters
    - [x] Integrate with backend
  - [x] Including mobile markup/implement markup

**Comment by danrminds** [comment]
Waiting for refinement

**Comment by jamalv** [comment]
Part of https://github.com/teamforus/forus/issues/1175

**Comment by jamalv** [comment]
Had a discussion about this, we decided not to wait on the new design and to implement this with the minimal required filters:
* Search
* Fund
* Category

**Comment by jamalv** [comment]
needs a plan

**Comment by jamalv** [comment]
@HardCoderNET please find the branch :)

**Comment by maxvisser** [comment]
![image](https://user-images.githubusercontent.com/10818702/71178246-0ee72e00-226e-11ea-8014-2481a27ad534.png)

**Comment by maxvisser** [comment]
![image](https://user-images.githubusercontent.com/10818702/71178295-2c1bfc80-226e-11ea-9b29-57e48aabab26.png)

**Comment by maxvisser** [comment]
![image](https://user-images.githubusercontent.com/10818702/71178326-3f2ecc80-226e-11ea-8826-1419c25229ca.png)

**Comment by jamalv** [comment]
For now, this issue is superseded by: https://github.com/teamforus/forus/issues/1339 and https://github.com/teamforus/forus/issues/1340

**Comment by maxvisser** [comment]
mobile design markup / implementing markup is not done yet.
